WooCommerce/Classes/ViewRelated/Coupons/CouponListViewController.swift

```diff
@@ -1,19 +1,20 @@
+import Combine
 import UIKit
 import WordPressUI
 
 final class CouponListViewController: UIViewController {
     @IBOutlet private weak var tableView: UITableView!
-    private var viewModel: CouponListViewModel!
+    private let viewModel: CouponListViewModel
 
     /// Set when an empty state view controller is displayed.
     ///
     private var emptyStateViewController: UIViewController?
 
+    private var subscriptions: Set<AnyCancellable> = []
```

Comment:
I see that we use both `subscriptions` and `cancellables` naming in the codebase, I wonder if it's better to settle on one 🤔 (I'm fine with either, just thought it's nice to be consistent)

Reply:
I think `subscriptions` makes it clearer what the property is about, which is why I chose it 😄 `cancellable` is more of a type name, and honestly for folks not familiar with Combine, it can be hard to understand what it is. WDYT?

Reply:
yea makes sense! it reads better when calling `cancel()` with `subscription.cancel()` than `cancellable.cancel()`. maybe we can ping the team about this naming convention in the channel?

Reply:
Sure, definitely!